app/models/metadata.rb

```diff
@@ -53,7 +53,26 @@ def self.filters_to_json
   end
 
   def self.metadata_to_json
-    Metadata.all.order(id: :asc).to_json
+    output = []
+    metadata = Metadata.all.order(id: :asc)
+    metadata.to_a.each do |meta|
+      meta_attributes = meta.attributes
+      meta_attributes[:themes] = []
+      meta_attributes[:themes] << "marine_spatial_planning" if meta_attributes["marine_spatial_planning"]
+      meta_attributes[:themes] << "education" if meta_attributes["education"]
+      meta_attributes[:themes] << "environmental_impact_assessment" if meta_attributes["environmental_impact_assessment"]
+      meta_attributes[:themes] << "ecosystem_assessment" if meta_attributes["ecosystem_assessment"]
+      meta_attributes[:themes] << "ecosystem_services" if meta_attributes["ecosystem_services"]
```

Comment:
You should be able to use some metaprogramming or loops to avoid all those manual assignments and deletions

Reply:
Thanks @Levia I will amend this code.